src/components/TodoAddForm.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+    <div class = 'addtodo div'>
+        <form @submit.prevent='onAddTodoFormSubmit()' > 
+            <label for="ftitle">Title</label>
+            <input type="text" id="ftitle" name="title" v-model="todoData.title" placeholder="title">
+            
+            <label for="fdescription">Description</label>
+            <input type="text" id="fdescription" name="description" v-model="todoData.description" placeholder="description">
+            
+            <label for="fprioriyt">Priority</label>
+            <select id='fprioriyt' v-model="todoData.priority">
+                <option value="0">Select priority:</option>
+                <option value="1">high</option>
+                <option value="2">medium</option>
+                <option value="3">low</option>
+
+            </select>
```

Comment:
e pazi sad ovo, imas ovde i ispod u komponenti identicno parce koda
aj napravi odvojenu komponentu za ovo koja dobija kroz props opcije koje prikazuje i kad se izabere nesto ispaljuje event, i onda gde koristis menjas todoData.priority na to sto ti ona emituje

ne bi trebalo da koristi v-model ta nova komponenta